docs/examples/02_qaoa.ipynb

```diff
@@ -220,15 +260,17 @@
    "source": [
     "serverless = ServerlessProvider(\n",
     "    token=os.environ.get(\"GATEWAY_TOKEN\", \"awesome_token\"),\n",
-    "    host=os.environ.get(\"GATEWAY_HOST\", \"http://localhost:8000\"),\n",
+    "    host=os.environ.get(\"GATEWAY_HOST\", \"http://localhost:59994\"),\n",
```

Comment:
this looks like the only code change needed to run the notebook? if so, I'd just make the update in the notebook but not rerun all the cells

Reply:
The only necessary change is `verify=False` in 
```
service = QiskitRuntimeService(
    channel='ibm_quantum',
    instance='ibm-q/open/main',
    token='<token>',
    verify=False,
)
```
I'll make it the only change and recheck in.
Thanks!